Automattic/jetpack
# Jetpack plugin: hide backup preparation message for sites without Backup

Fixes 1164141197617539-as-1201311518372705

#### Changes proposed in this Pull Request:
Right after setting up a brand new site with Jetpack, the _At a Glance_ section shows a "backup in preparation" message even if the site is not provisioned with Backup.

This PR fixes this.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

- Set up a new self-hosted site with this branch as the plugin version
- In WP admin, activate Jetpack
- In the pricing page section, select _Free_
- Back in WP admin, visit the _Jetpack > At a Glance_ section
- In the _Backup_ panel, check that you see an upsell for Backup
<img width="501" alt="Screen Shot 2021-12-07 at 10 28 17 AM" src="https://user-images.githubusercontent.com/1620183/145057954-46fe8879-ce81-428c-9539-6b3215f0699f.png">

- Now purchase a subscription that includes Backup
- In the _At a Glance_ section, you should now see the preparing backup message
<img width="502" alt="Screen Shot 2021-12-07 at 10 29 06 AM" src="https://user-images.githubusercontent.com/1620183/145058149-6bbffceb-abbc-4ca8-bcb4-f645675de3c4.png">

- 5 minutes after the site has been created, you should see the server credentials call-to-action
<img width="499" alt="Screen Shot 2021-12-07 at 10 31 21 AM" src="https://user-images.githubusercontent.com/1620183/145058425-fce43be0-52a0-4391-b0de-4d67c96000b5.png">
